pipeline/coordinator.go

```diff
@@ -145,6 +146,10 @@ func (j *JobInfo) ReportProgress(stage clients.TranscodeStatus, completionRatio
 	_ = j.statusClient.SendTranscodeStatus(tsm)
 }
 
+func ClippingRetryBackoff() backoff.BackOff {
```

Comment:
This ties us up for quite a while in terms of job capacity. Is there a more intelligent method we can move to in the future? Maybe having Task Runner handle the retries?

Reply:
True, we can do that. Let me file a ticket and we'll follow up quickly after launch to address this.